cmd/konvoy-image/cmd/create-package-bundle.go

```diff
@@ -0,0 +1,75 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	packageCmd        = "create-package-bundle"
+	packageCmdExample = "create-package-bundle --os redhat-8.4 --kubernetes-version=1.28.6 --output-directory=artifacts"
+	validOS           = []string{
+		"centos-7.9",
+		"redhat-7.9",
+		"redhat-8.4",
+		"redhat-8.6",
+		"redhat-8.8",
+		"oracle-7.9",
```

Comment:
Are we missing the code to build `oracle-7.9`?

Reply:
nice catch. this should be removed, we're not implementing this.